enforcer/src/main/java/org/wso2/micro/gateway/enforcer/subscription/SubscriptionDataStoreImpl.java

```diff
@@ -313,13 +312,6 @@ public void addOrUpdateAPI(API api) {
 
     @Override
     public void addOrUpdateAPIWithUrlTemplates(API api) {
```

Comment:
do we need these methods in the interface?

Reply:
This is the exact interface used in API Manager. Hence we maintain the same but keep the body as unimplemented.